### PR TITLE
Gnome notification fix

### PR DESCRIPTION
**Pull request by Djaler**
Due to Gnome Guidelines https://wiki.gnome.org/Initiatives/GnomeGoals/NotificationSource

Now, due to the absence of these parameters, notifications from the telegram are displayed as "Others"

### CODE REVIEW

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp [hunk 1]
```diff
@@ -378,6 +378,7 @@ void RegisterCustomScheme() {
 			s << "Type=Application\n";
 			s << "Categories=Network;InstantMessaging;Qt;\n";
 			s << "MimeType=x-scheme-handler/tg;\n";
+			s << "X-GNOME-UsesNotifications=true\n";
```
Comment (tiendq) [inline Telegram/SourceFiles/platform/linux/specific_linux.cpp]: Missing trailing `;`?

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp [hunk 2]
```diff
@@ -378,6 +378,7 @@ void RegisterCustomScheme() {
 			s << "Type=Application\n";
```
Comment (tiendq) [inline Telegram/SourceFiles/platform/linux/specific_linux.cpp]: Missing trailing `;`?

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp (hunk 1 above)
Comment (tiendq) [inline Telegram/SourceFiles/platform/linux/specific_linux.cpp]: Missing trailing `;`?

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp (hunk 2 above)
Comment (tiendq) [inline Telegram/SourceFiles/platform/linux/specific_linux.cpp]: Missing trailing `;`?

File: Telegram/SourceFiles/platform/linux/specific_linux.cpp (hunk 1 above)
Comment (Djaler) [reply Telegram/SourceFiles/platform/linux/specific_linux.cpp]: No, `;` is needed in some sections, like a `MimeType` or `Categories`

**Comment by stek29** [comment]
@Djaler @tiendq can't accept the pr :)

**Comment by Djaler** [comment]
So, @john-preston, can you?

**Pull request merged**